linode_api4/paths/lke.py

```diff
@@ -0,0 +1,40 @@
+from linode_api4.objects.lke import TieredKubeVersion
+from linode_api4.paths.api_path import ApiPath
+
+
+class LKETier(ApiPath):
+    """
+    The API path for a single LKE tier.
+    """
+
+    def __init__(
+        self,
+        client: "LinodeClient",
+        id: str,
+    ):
+        self.client = client
+        self.id = id
+
+    @property
+    def endpoint(self) -> str:
+        return f"/lke/tiers/{self.id}"
+
+    def versions(self, *filters):
+        """
+        Returns a paginated list of versions for this tier matching the given filters.
+
+        API Documentation: Not Yet Available
+
+        :param filters: Any number of filters to apply to this query.
+                        See :doc:`Filtering Collections</linode_api4/objects/filtering>`
+                        for more details on filtering.
+
+        :returns: A paginated list of kube versions that match the query.
+        :rtype: PaginatedList of TieredKubeVersion
+        """
+
+        return self.client._get_and_filter(
+            TieredKubeVersion,
+            endpoint=f"{self.endpoint}/versions",
+            *filters,
+        )
```

Comment:
Any thoughts on this new pattern? IMO it feels a bit over-engineered but I thought it might make sense because the change from `/lke/versions/standard` and `/lke/versions/enterprise` to `/lke/tiers/{tier}/versions` was deliberate, implying they might extend the path in the future.

The usage with this pattern will look like:

```
LinodeClient().lke.tier("standard").versions()
```

and the legacy-style usage pattern would look like:

```
LinodeClient(...).lke.versions_for_tier("standard")
```

If anyone else has any opinions on this please let me know! I'm happy to revert it if we think it's overkill.